Missionframework/KPGUI/KPGUI_defines.hpp

```diff
@@ -142,6 +142,16 @@
 #define KP_WIDTH_VAL_LP                 0.2
 #define KP_HEIGHT_VAL_LP                (1 - 2 * KP_Y_VAL_LP - KP_HEIGTH_TITLE - KP_HEIGTH_BUTTON - 2 * KP_SPACING_Y)
 
+/*
+    --- Right panel dialog components ---
+*/
+
+#define KP_X_VAL_RP                     0.8 - 0.0025
+#define KP_Y_VAL_RP                     0.0025
+
+#define KP_WIDTH_VAL_RP                 0.2
+#define KP_HEIGHT_VAL_RP                (1 - 2 * KP_Y_VAL_RP - KP_HEIGTH_TITLE - KP_HEIGTH_BUTTON - 2 * KP_SPACING_Y)
```

Comment:
```suggestion
#define KP_HEIGHT_VAL_RP                (0.905 - KP_HEIGTH_TITLE - KP_HEIGTH_BUTTON - 2 * KP_SPACING_Y)
```
same for left panel dialog